app/content/management/commands/migrate_members.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand
+from django.db.transaction import atomic
+
+from app.content.models import User
+from app.content.models.user import CLASS
+from app.group.models import Group, Membership
+
+STUDY_FULL = (
+    (1, "Dataingeniør"),
+    (2, "Digital forretningsutvikling"),
+    (3, "Digital infrastruktur og cybersikkerhet"),
+    (4, "Digital samhandling"),
+    (5, "Drift"),
+    (6, "Informasjonsbehandling"),
+)
+
+
+class Command(BaseCommand):
+    args = ""
+    help = "Migrate users to class and study groups"
+
+    def handle(self, *args, **options):
+        self.migrate_year()
+        self.migrate_study()
+
+    @atomic
+    def migrate_year(self):
+        this_year = 2022
+
+        for year, _ in CLASS:
+
+            users = User.objects.filter(user_class=year)
+            if year == -1:
+                group_name = "Alumni"
```

Comment:
Hva tenker du om å sette de som er alumni til 2018? De fleste er fra da

Reply:
Er vel en så liten andel det ikke gjelder for at jeg tror vi kan gjøre det med relativt stor sikkerhet